src/StackExchange.Redis/PerfCounterHelper.cs

```diff
@@ -77,6 +77,13 @@ internal static int GetThreadPoolStats(out string iocp, out string worker)
 
             iocp = $"(Busy={busyIoThreads},Free={freeIoThreads},Min={minIoThreads},Max={maxIoThreads})";
             worker = $"(Busy={busyWorkerThreads},Free={freeWorkerThreads},Min={minWorkerThreads},Max={maxWorkerThreads})";
+
+#if NETCOREAPP
+            workItems = $"(Threads={ThreadPool.ThreadCount},QueuedItems={ThreadPool.PendingWorkItemCount},CompletedItems={ThreadPool.CompletedWorkItemCount})";
```

Comment:
how often is this called? should I care about the `string` alloc?

Reply:
I'm not a big fan of how existing works - was light touch, let's get existing PRs touching it in, then I'll add a bench and do a builder pass and such on the exception path, no reason we can't be civilized with the GC!